pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -301,6 +304,10 @@ func (self *AWSCloud) AddSSHKeyToAllInstances(user string, keyData []byte) error
 }
 
 func (a *AWSCloud) CurrentNodeName(hostname string) (string, error) {
+	if a.cfg.Global.UseKubernetesNodeTag {
```

Comment:
We'd have to do a `DescribeTags` call I believe to get it. Conceptually it's better but it is a bit more code and an additional aws api call.